docs/source/examples/Variable Inspector.ipynb

```diff
@@ -80,10 +80,8 @@
     "            '</td></tr><tr><td>'.join(['{0}</td><td>{1}</td><td>{2}'.format(v, type(eval(v)).__name__, str(eval(v))) for v in values]) + \\\n",
     "            '</td></tr></table></div>'\n",
     "\n",
-    "    def _ipython_display_(self):\n",
-    "        \"\"\"Called when display() or pyout is used to display the Variable \n",
-    "        Inspector.\"\"\"\n",
-    "        self._box._ipython_display_()\n"
+    "    def _repr_mimebundle_(self, **kwargs):\n",
+    "        return self._box._repr_mimebundle_(**kwargs)"
```

Comment:
I need to remove the `_ipython_display_` call because if was removed from the base Jupyter widget.